core/core.go

```diff
@@ -86,6 +87,10 @@ const (
 	onlineMode
 )
 
+func init() {
+	identify.ClientVersion = "go-ipfs/" + config.CurrentVersionNumber + "/" + config.CurrentCommit
```

Comment:
Yeah, I think thats fine. Makes it more apparent the code was built without a commit